src/app/machines/views/MachineDetails/MachineHeader/MachineHeader.test.tsx

```diff
@@ -48,10 +48,10 @@ describe("MachineHeader", () => {
   it("displays a spinner when loading", () => {
     state.machine.items = [];
 
-    renderWithBrowserRouter(
-      <MachineHeader setSidePanelContent={vi.fn()} systemId="abc123" />,
-      { state, route: "/machine/abc123" }
-    );
+    renderWithBrowserRouter(<MachineHeader systemId="abc123" />, {
```

Comment:
This file's tests need to be migrated 

```suggestion
    renderWithProviders(<MachineHeader systemId="abc123" />, {
```